.gitignore

```diff
@@ -1,3 +1,5 @@
 .pio
 .vscode
 *_env.ini
+main\User_config.h
+main/User_config.h
```

Comment:
I think we should remove this modification, indeed User_config.h needs to be stored on the configuration management.
If you have personal keys-password you can store them in a _env.ini file with platformio.

Reply:
Whoops, meant to take that back out before the pull request. Thanks for the tip about the _env.ini, I'll try that. This is my first time using platformio.